samples/bot-formatting-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,131 @@
+---
+page_type: sample
+description: This sample demonstrates the use of different formatting on cards
```

Comment:
This sample shows how to use different formatting on adaptive card.

Reply:
Fixed